package.json

```diff
@@ -51,6 +51,7 @@
   "devDependencies": {
     "@types/node": "^6.0.45",
     "babel-core": "^6.16.0",
+    "browserify-versionify": "^1.0.6",
```

Comment:
I believe we also wanted this inside the report, or is this really only extension?